packages/sdk/package.json

```diff
@@ -13,9 +13,17 @@
   },
   "main": "dist/index.js",
   "module": "dist/index.js",
+  "types": "dist/index.d.ts",
   "files": [
     "dist"
   ],
+  "exports": {
+    ".": {
+      "types": "dist/index.d.ts",
```

Comment:
I don't think we need here cuz line 16 already specified the entry point for types